tests/test-process.cpp

```diff
@@ -91,11 +91,11 @@ void test_r_file() {
 
     process p = process_builder().command(SHELL)
 #ifndef MOZART_PLATFORM_WIN32
-        .environment("VAR", "fuckcpp")
+                    .environment("VAR", "fuckcpp")
 #endif
-        .redirect_stdout(fileno(fout))
-        .merge_outputs(true)
-        .start();
+            .redirect_stdout(fileno(fout))
+            .merge_outputs(true)
+            .start();
```

Comment:
Here breaks the indent level.
```
                   .environment("VAR", "fuckcpp")
```
should be idented in the same level as the following
```
            .redirect_stdout(fileno(fout))
            .merge_outputs(true)
            .start();
```